content/zh/blog/kubeedge-deployment-manual/index.md

```diff
@@ -0,0 +1,569 @@
+
+**编译kubeadm**
+```
+[root@ke-cloud ~]# cd $GOPATH/src/github.com/kubeedge/kubeedge
+[root@ke-cloud ~]# make all WHAT=keadm
+
+说明：编译后的二进制文件在./_output/local/bin下，单独编译cloudcore与edgecore的方式如下：
+[root@ke-cloud ~]# make all WHAT=cloudcore && make all WHAT=edgecore
+```
+
+**创建cloud节点**
+```
+[root@ke-cloud ~]# keadm init --advertise-address="192.168.1.66"
+
+Kubernetes version verification passed, KubeEdge installation will start...
+
+...
+
+KubeEdge cloudcore is running, For logs visit:  /var/log/kubeedge/cloudcore.log
+CloudCore started
+```
+
+## 3.2 edge端配置
+edge端也可以通过keadm进行配置，可以将cloud端编译生成的二进制可执行文件通过scp命令复制到edge端。
+
+### 3.2.1 从云端获取令牌
+在云端运行将返回令牌，该令牌将在加入边缘节点时使用。`keadm gettoken`
+```
+[root@ke-cloud ~]# keadm gettoken
+8ca5a29595498fbc0648ca59208681f9d18dae86ecff10e70991cde96a6f4199.eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJleHAiOjE1OTUwMzU0Mjh9.YR-N628S5wEFLifC0sM9t-IuIWkgSK-kizFnyAy5Q50
+```
+
+### 3.2.2 加入边缘节点
+`keadm join`将安装edgecore和mqtt。它还提供了一个标志，通过它可以设置特定的版本。
+```
+[root@ke-edge1 ~]# keadm join --cloudcore-ipport=192.168.1.66:10000 --token=8ca5a29595498fbc0648ca59208681f9d18dae86ecff10e70991cde96a6f4199.eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJleHAiOjE1OTUwMzU0Mjh9.YR-N628S5wEFLifC0sM9t-IuIWkgSK-kizFnyAy5Q50
+
+Host has mosquit+ already installed and running. Hence skipping the installation steps !!!
+
+...
+
+KubeEdge edgecore is running, For logs visit:  /var/log/kubeedge/edgecore.log
+```
+重要的提示：
+* --cloudcore-ipport 标志是强制性标志。
+* 如果要自动为边缘节点应用证书，--token则需要。
+* 云和边缘端使用的kubeEdge版本应相同。
+
+## 3.3 验证
+边缘端在启动edgecore后，会与云端的cloudcore进行通信，K8s进而会将边缘端作为一个node纳入K8s的管控。
+```
+[root@ke-cloud ~]# kubectl get node
+NAME       STATUS   ROLES        AGE   VERSION
+ke-cloud   Ready    master       13h   v1.17.0
+ke-edge1   Ready    agent,edge   64s   v1.17.1-kubeedge-v1.3.1
+
+[root@ke-cloud ~]# kubectl get pods -n kube-system
+NAME                               READY   STATUS    RESTARTS   AGE
+coredns-6955765f44-k65xx           1/1     Running   0          13h
+coredns-6955765f44-r2q6g           1/1     Running   0          13h
+etcd-ke-cloud                      1/1     Running   0          13h
+kube-apiserver-ke-cloud            1/1     Running   0          13h
+kube-controller-manager-ke-cloud   1/1     Running   0          13h
+kube-flannel-ds-amd64-fgtdq        0/1     Error     5          5m55s
+kube-flannel-ds-amd64-lrsrh        1/1     Running   0          13h
+kube-proxy-vr44d                   1/1     Running   0          13h
+kube-scheduler-ke-cloud            1/1     Running   0          13h
+
+说明：如果在K8s集群中配置过flannel网络插件（见2.7），这里由于edge节点没有部署kubelet，
+所以调度到edge节点上的flannel pod会创建失败。这不影响KubeEdge的使用，可以先忽略这个问题。
+```
+
+# 四、运行KubeEdge示例
+**选用示例：KubeEdge Counter Demo**
+计数器是一个伪设备，用户无需任何额外的物理设备即可运行此演示。计数器在边缘侧运行，用户可以从云侧在Web中对其进行控制，也可以从云侧在Web中获得计数器值。原理图如下：
+
+![image](https://github.com/Poorunga/website/blob/wangjiezhang/content/zh/blog/kubeedge-deployment-manual/images/work-flow.png)

```

Comment:
Keep the images in the same folder or place them in the `static/img` folder and reference them using the `figure` shortcode like this:
`{{< figure src="<file_name>" title="your_title_here" >}}`
You can use the markdown way to reference images as well (as you have done) but no need to use absolute URLs unless the images are being served off a different server.